README.rst

```diff
@@ -80,6 +80,7 @@ you don’t have them, some features will not be working.
 -  `RTMPDump`_ 2.4 or higher to download RTMP streams.
 -  `PyCrypto`_ to download encrypted HLS streams
 -  `Requests`_
+-  `PySocks`_ to enable proxy support
```

Comment:
You want to add a link (see the bottom of the document) for this to be rendered correctly

Reply:
Oups! Fixd!